opa/models/session.py

```diff
@@ -0,0 +1,60 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class Session(models.Model):
+    _name = 'opa.session'
+    _description = 'Session from an Open Academy course'
+
+    name = fields.Char(string='Title', required=True)
```

Comment:
Remove string and rename your variable as title

```suggestion
    title = fields.Char(required=True)
```